### ISSUE TITLE
TypeError: convertapi_1.default is not a function

### ISSUE THREAD

**Issue by danielzev** [ConvertAPI/convertapi-node#35]
My NodeJS/Typescript code looks like this:

```
import convertapiPackage from 'convertapi';

...

const convertapi = convertapiPackage('MY_SECRET');
return convertapi
    .convert('pdf', {
        File: url,
    })
    .then(function (result) {
        // stuff here
    })
    .catch(function (e) {
        // stuff here
    });
```

When I ran than I was getting an error

```
TypeError: convertapi_1.default is not a function
```

After reading on some posts online, people had this generic issue:

https://stackoverflow.com/questions/61524130/typescript-babel-import-causing-1-default-is-not-a-function

The recommendation is to update your `tsconfig.json` file with `"esModuleInterop": true`

Once I did that, I had to update a bunch of imports in my code, but once I got through that, it worked.

Anyway, why does the convertApi NodeJS SDK enforce us to set `"esModuleInterop": true`?

**Comment by laurynas-baltsoft** [comment]
@danielzev thanks for the report.
Added `ConvertAPI` class export and released `1.10.0` client version. You can import like this:

```javascript
import { ConvertAPI } from 'convertapi';

const convertapi = new ConvertAPI('your-api-secret');
```

It will not require `esModuleInterop` option.

**Comment by danielzev** [comment]
Love it! thanks @laurynas-baltsoft
